.ci/docker/Dockerfile

```diff
@@ -44,13 +44,13 @@ WORKDIR /
 RUN apt-get remove -y $DEPS_IVERILOG \
  && rm -Rf ghdl-0.36 iverilog-10_3 \
  && apt-get autoremove -y --purge \
- && apt-get clean \
  && rm /etc/apt/apt.conf.d/docker-clean \
- && apt-get install -y --download-only \
+ && apt-get install -y \
```

Comment:
I was entirely guided by CI speeds when making this change, extracting the packages takes a relatively long time. Seeing docker images are cached, I don't think size matters that much. What's your hesitation?

Reply:
My worry is more that something changing in an early layer of the image could cause runner time to balloon up, as every layer after would be invalidated (AFAIK; I'm not aware of any particularly smart caching in docker) and it would have to download a lot of new layers

Reply:
Wait can that happen? I thought layers are linked together using their sha256 hashes.

Reply:
If the cache can't be used for one layer, it isn't used for all subsequent layers

Reply:
Ah alright, I'm not too worried. We change the Docker image _maybe_ four times a year.